_source/reference/frames.md

```diff
@@ -139,7 +139,10 @@ attributes and JavaScript properties.
 All `<turbo-frame>` elements can be controlled in JavaScript environments
 through instances of the `FrameElement` class.
 
-* `FrameElement.src` controls the pathname or URL to be loaded
+* `FrameElement.src` controls the pathname or URL to be loaded. Setting the `src` 
+   property will immediately navigate the element. When `FrameElement.loaded` is 
```

Comment:
Should `FrameElement.loaded` reference the [FrameElement.loading](https://turbo.hotwired.dev/reference/frames#html-attributes) attribute instead of the `.loaded` property?